build.gradle.kts

```diff
@@ -99,3 +93,12 @@ subprojects {
         }
     }
 }
+
+fun getPackageVersion(projectName: String): String {
```

Comment:
In the future, we might consider adding to this function with part of [this logic](https://github.com/WordPress/gutenberg/blob/cb96d5946d677d6e30b692cbbcdd6603eb57d9fd/packages/react-native-bridge/android/extractPackageVersion.gradle), in order to support dependencies from forked repositories.